src/main/native/windows/util.cc

```diff
@@ -92,20 +92,31 @@ bool AutoAttributeList::Create(HANDLE* handles, size_t count,
     return false;
   }
 
+  std::unique_ptr<AutoAttributeList> attr_list(
+      new AutoAttributeList(std::move(data), stdin_h, stdout_h, stderr_h));
+  static constexpr size_t kHandleCount = 3;
```

Comment:
Just one small thing: if you want to use `kHandleCount` to avoid magic number, consider moving this class declaration so that `StdHandles` can use it too.